src/main/java/uk/gov/hmcts/reform/sendletter/tasks/reports/EmailSender.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.hmcts.reform.sendletter.tasks.reports;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.mail.MailException;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.MimeMessageHelper;
+import org.springframework.stereotype.Component;
+
+import javax.mail.MessagingException;
+import javax.mail.internet.MimeMessage;
+
+@Component
+@ConditionalOnProperty(prefix = "spring.mail", name = "host")
+public class EmailSender {
+
+    private static final Logger log = LoggerFactory.getLogger(EmailSender.class);
+
+    static final String EMAIL_BODY = "This is an auto generated email. Do not respond to it.";
+
+    private final JavaMailSender mailSender;
+    private final String from;
+
+    public EmailSender(
+        JavaMailSender mailSender,
+        @Value("${spring.mail.username}") String from
+    ) {
+        this.mailSender = mailSender;
+        this.from = from;
+    }
+
+    public void send(
+        String subject,
+        String[] recipients,
+        Attachment... attachments
+    ) {
+        try {
+            MimeMessage message = mailSender.createMimeMessage();
```

Comment:
Adding info logs here would be nice- before and after the email has been sent

Reply:
Yep, good point. Was thinking few moments ago about it. Just added a basic `debug` kind of log. The actual `info`s are awaiting on actual report